pkg/sql/ttl/ttljob/ttljob_test.go

```diff
@@ -127,6 +129,9 @@ func newRowLevelTTLTestJobTestHelper(
 func (h *rowLevelTTLTestJobTestHelper) waitForScheduledJob(
 	t *testing.T, expectedStatus jobs.Status, expectedErrorRe string,
 ) {
+	h.env.SetTime(timeutil.Now().Add(time.Hour * 24))
```

Comment:
These 2 lines were always called before `waitForScheduledJob` so I'm inlining them here to reduce duplication.